hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -2167,7 +2280,34 @@ public String getCanonicalServiceName() {
    * @throws IOException Other IO problems
    */
   private void copyFile(String srcKey, String dstKey, long size)
-      throws IOException, InterruptedIOException, AmazonClientException {
+      throws IOException, AmazonClientException {
```

Comment:
leave as before so as to make clear it may be interrupted, unless its mentioned in the javadocs

Reply:
Fixed